HST/product_labels/__init__.py

```
@@ -651,45 +655,43 @@ def find_suffix_dicts(ipppssoot_dict, suffixes):
-            if reference_suffixes:
-                break
-
-        # If that fails, check the ALT_REF_SUFFIXES list
-        if not reference_suffixes:
-            tag = 'Alternative reference'
+        # Search for a reference suffix among the available suffixes.
+        # There are three levels of suffix precedence defined in `suffix_info`; lower
+        # precedence suffixes are only used if all higher-precedence suffixes are absent.
+        # `tag` is one of "Reference", "Alternative Reference", or "Second Alt Reference".
+        reference_suffixes = []
+        for tag, suffixes in zip(suffix_info.REF_TAGS, suffix_info.REF_SUFFIXES):
```

Ruff (0.14.4)

Loop control variable `tag` not used within loop body

(B007)